nirc_ehr/src/org/labkey/nirc_ehr/table/NIRC_EHRCustomizer.java

```diff
@@ -42,13 +43,121 @@ public void customize(TableInfo table)
     {
         if (table instanceof AbstractTableInfo ti)
         {
+            sharedCustomization(ti);
             doSharedCustomization(ti);
```

Comment:
looks like some columns like "performedBy" and "gender" are getting customized both in sharedCustomization & doSharedCustomization and customizations in sharedCustomization may get lost. Can these two methods be consolidated?

Reply:
Good catch! now consolidated